qt/TorrentDelegate.cc

```diff
@@ -426,6 +419,14 @@ TorrentDelegate::setProgressBarPercentDone (const QStyleOptionViewItem & option,
 }
 
 void
+TorrentDelegate::applyColor(const QColor& highlight, const QColor& base, const QColor& window) const
```

Comment:
Mike,

Thanks for all the responses, I'll redo the patch series taking into
account everything you said.

Tomaz

On Wed, Mar 2, 2016 at 8:51 AM, Mike Gelfand <notifications@github.com>
wrote:

> In qt/TorrentDelegate.cc
> <https://github.com/transmission/transmission/pull/7#discussion_r54713228>
> :
>
> > @@ -426,6 +419,14 @@ TorrentDelegate::setProgressBarPercentDone (const QStyleOptionViewItem & option,
> >  }
> >
> >  void
> > +TorrentDelegate::applyColor(const QColor& highlight, const QColor& base, const QColor& window) const
>
> Yet again, nothing to do with Qt 5 or C++11.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/transmission/transmission/pull/7/files#r54713228>.
>